.plzconfig

```diff
@@ -0,0 +1,30 @@
+[Plugin "python"]
+# Use plugin's defaults
+
+[Plugin "proto"]
+ProtocTool = //third_party/proto:protoc
+LanguageDef = //build_defs:python
+
+[Plugin "python_proto"]
+# Use this plugin's defaults
+
+[PluginDefinition]
+Name = python_proto
+
+[PluginConfig "plugin"]
+ConfigKey = ProtoPlugin
+DefaultValue = grpc_python_plugin
+Inherit = true
+
+[PluginConfig "proto_dep"]
+ConfigKey = ProtoDep
+DefaultValue = //third_party/python:protobuf
```

Comment:
Actually, this should reference the target relative to the plugin, not on the host one. I didn't include the `@self` here since that will be obsolete. The `Inherit = true` below will let the user override it.